js/task03.js

```diff
@@ -0,0 +1,17 @@
+const item1 = {};
+item1.href = "a1.html";
+item1.src = "/img/bookmark.png";
+item1.text = "a1";
+
+const item2 = {};
+item2.href = "a2.html";
+item2.src = "/img/bookmark.png";
+item2.text = "a2";

```

Comment:
あレ。これはなんで一回で初期化しなかったですか？
```suggestion
const item1 = {};
item1.href = "a1.html";
item1.src = "/img/bookmark.png";
item1.text = "a1";

const item2 = {
href: "a2.html",
src: "/img/bookmark.png",
text: "a2"
};

```

Reply:
ご指摘いただいた通り、1回でオブジェクトを作るように変更します。